static/css/budget.less

```diff
@@ -11,30 +17,30 @@
 
       li {
           .list_item();
-          
+
           a {
               .list_link();
               .jump-link();
           }
       }
     }
-    
-    &.financial-report a, 
+
+    &.financial-report a,
     &.performance-plan a,
     &.funding-request a span{
         &:before {
             .cf-icon();
-            content: "\e402";
+            content: "\e406";
```

Comment:
@Scotchester the `.cf-icon-download()` adds it's own `:before` so it should work but only if you add it to the parent `span`. Not sure what that would do to the `.cf-icon` being called within the `:before` though. Seems it might over-write some things.